classes/genre.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'item'
+
+class Genre
+  attr_accessor :name, :items
+  attr_reader :id
+
+  def initialize(name)
+    @id = rand(1...999)
+    @name = name
+    @items = []
+  end
+
+  def add_item(items)
+    case items
+    when Item
+      items.add_genre(self)
+      @items << items
+    else
```

Comment:
- [ ] I am unable to add a genre successfully, the program takes in input and upon enering the last value, loops back to the main menu, kindly fix this issue